### PR TITLE
Use DIE for error handling

### PR DESCRIPTION
**Pull request by Alex-deVis**

### CODE REVIEW

File: content/chapters/app-interact/lab/support/time-server/server.c [hunk 1]
```diff
@@ -7,49 +8,36 @@
 #include <time.h>
 #include <unistd.h>
 
+#include "../utils/utils.h"
+
 #define BIND_ADDR "0.0.0.0"
 #define PORT 2000
 
-int create_socket(char *addr, short port)
+int create_socket(char *addr)
```
Comment (molecula2788) [inline content/chapters/app-interact/lab/support/time-server/server.c]: I think the port should be left as function argument.

File: content/chapters/app-interact/lab/content/time-server.md [hunk 2]
```diff
@@ -10,15 +10,15 @@ The client will then print the received time to the console.
 Let's build and run this example
 
 ```
-student@os:~/.../lab/time-server$ make
+student@os:~/.../support/time-server$ make
 gcc -Wall -o server server.c
 gcc -Wall -o client client.c
-student@os:~/.../lab/time-server$ ./server
+student@os:~/.../support/time-server$ ./server 2000
```
Comment (molecula2788) [inline content/chapters/app-interact/lab/content/time-server.md]: The server doesn't use the command line arguments (like `2000`).

File: content/chapters/app-interact/lab/support/time-server/server.c (hunk 1 above)
Comment (Alex-deVis) [reply content/chapters/app-interact/lab/support/time-server/server.c]: Commited an unintended change.

File: content/chapters/app-interact/lab/content/time-server.md (hunk 2 above)
Comment (Alex-deVis) [reply content/chapters/app-interact/lab/content/time-server.md]: Commited an unintended change.

**Pull request merged**